lib/devise.rb

```diff
@@ -153,7 +153,11 @@ module Test
   mattr_accessor :pepper
   @@pepper = nil
 
-  # Used to enable sending notification to user when their password is changed
+  # Used to send notification to the original user email when their email is changed.
+  mattr_accessor :send_email_change_notification
```

Comment:
Yeah it probably makes sense. I considered that but kept `email_change` for consistency with `password_change`. 

There's also the scenario with reconfirmable where the change hasn't actually happened yet, but I guess `email_changed` would work too.

If we go with `email_changed`, I'd vote for deprecating `password_change` in favor of `password_changed` - just not sure the work is worth it :smile: